dev-util/flatpak-builder/flatpak-builder-1.4.3.ebuild

```diff
@@ -20,7 +22,9 @@ RDEPEND="
 	>=dev-libs/libxml2-2.4:=
 	dev-libs/json-glib:=
 	net-misc/curl:=
+	dev-util/debugedit:=
 	yaml? ( dev-libs/libyaml:= )
+	dev-libs/appstream[compose(+)]
```

Comment:
This should be `compose(-)` because it was disabled by default before, right?

Reply:
We explicitly need compose support. I believe this is how to specify it.

Reply:
What @zmedico said.

Reply:
@zmedico do you have anything in addition to say wrt dependency order? Usually we have alphanumeric sorting with optional deps below unconditional block, but some maintainers have their own ideas ...

Reply:
I like alphanumeric sorting with optional deps below unconditional block, so yeah let's do that.